src/protocols/secure_channel/tests/BUILD.gn

```diff
@@ -8,11 +8,15 @@ import("${chip_root}/build/chip/chip_test_suite.gni")
 chip_test_suite("tests") {
   output_name = "libSecureChannelTests"
 
-  test_sources = [ "TestStatusReport.cpp" ]
+  test_sources = [
+    "TestPASESession.cpp",
+    "TestStatusReport.cpp",
+  ]
 
   public_deps = [
     "${chip_root}/src/lib/core",
     "${chip_root}/src/lib/support",
+    "${chip_root}/src/messaging/tests:helpers",
```

Comment:
Note: You can depend on 

`"${chip_root}/src/messaging/tests_lib",`

provided you don't split these out into src/messaging/tests:helpers

Reply:
(I'm not saying this is wrong, either way works. Actually if you do that the messaging tests come along for the ride, although they will not end up being linked)